### PR TITLE
Student edit signup dn

### PR DESCRIPTION
**Pull request by dbnorth**

### CODE REVIEW

**Review by JuliaFlores21 — CHANGES_REQUESTED** [review]
There were four mistakes that I found and added to Agility.

File: src/components/student/StudentEventSignupEditDialog.vue [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+const props = defineProps({
+  eventData: { type: [Object], required: true },
+  eventSignUpData: { type: [Object], required: true },
+  studentInstrumentSignupData: { type: [Object], required: true },
+});
+const studentInstrumentSignup = ref(
+  Object.assign({}, props.studentInstrumentSignupData)
+);
+const loginStore = useLoginStore();
+console.log(loginStore);
```
Comment (JuliaFlores21) [inline src/components/student/StudentEventSignupEditDialog.vue:25]: we could delete this console.log

File: src/components/student/StudentEventSignupEditDialog.vue [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+          fullName = studentPiece.piece.composer.firstName;
+        }
+        studentPiece.piece.composer.fullName = fullName;
+      });
+      studentInstrumentSignup.value.eventSignup.eventSignupPieces.forEach(
+        (studentPiece) => {
+          selectedStudentPieces.value.push(studentPiece);
+        }
+      );
+      console.log(selectedStudentPieces.value);
```
Comment (JuliaFlores21) [inline src/components/student/StudentEventSignupEditDialog.vue:87]: we could also delete this console.log

File: src/components/student/StudentEventSignupEditDialog.vue [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+      console.log(e);
+    });
+}
+
+async function saveSignup() {
+  const studentInstrumentSignupData = {
+    id: studentInstrumentSignup.value.eventSignupId,
+    isGroupEvent: groupSignup.value,
+  };
+  console.log(groupSignup.value);
```
Comment (JuliaFlores21) [inline src/components/student/StudentEventSignupEditDialog.vue:140]: delete this console.log

**Review by JuliaFlores21 — CHANGES_REQUESTED** [review]

File: src/components/student/StudentEventSignupEditDialog.vue (hunk 2 above)
Comment (dbnorth) [reply src/components/student/StudentEventSignupEditDialog.vue:87]: done

File: src/components/student/StudentEventSignupEditDialog.vue (hunk 3 above)
Comment (dbnorth) [reply src/components/student/StudentEventSignupEditDialog.vue:140]: done

File: src/components/student/StudentEventSignupEditDialog.vue (hunk 1 above)
Comment (dbnorth) [reply src/components/student/StudentEventSignupEditDialog.vue:25]: done

**Review by arrianNaomi — APPROVED** [review]
looks good (was asked to do this ;))

**Pull request merged**